packages/nouns-contracts/contracts/governance/data/NounsDAOData.sol

```diff
@@ -117,7 +132,18 @@ contract NounsDAOData is OwnableUpgradeable, UUPSUpgradeable, NounsDAODataEvents
         string memory slug,
         uint256 proposalIdToUpdate
     ) external payable {
-        if (!isNouner(msg.sender) && msg.value < createCandidateCost) revert MustBeNounerOrPaySufficientFee();
+        if (proposalIdToUpdate > 0) {
+            INounsDAO dao = INounsDAO(nounsDao);
+            if (dao.state(proposalIdToUpdate) != NounsDAOStorageV3.ProposalState.Updatable)
```

Comment:
@eladmallel for gas purposes, maybe instead of calling `dao.state()`, you can check the `updatePeriodEndBlock` returned in the `propInfo` below?

Reply:
I agree, pushed a fix